### PR TITLE
docs(readme): update readme with npm and build badges

### PR DESCRIPTION
**Pull request by JamesSingleton**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,4 +49,4 @@ Any contributions made under this project will be governed by the [Apache Licens
 ## 🗣️ Code of Conduct
 
 This project adheres to the [American Express Community Guidelines](./CODE_OF_CONDUCT.md).
-By participating, you are expected to honor these guidelines.
\ No newline at end of file
+By participating, you are expected to honor these guidelines.
```
Comment (mtomcal) [inline README.md]: Why did this change?

Comment (JamesSingleton) [reply README.md]: I have no idea. I don't remember touching that. I will add that back.

File: packages/holocron/README.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # holocron
```
Comment (jsmartfo) [inline packages/holocron/README.md]: no brand for the children? :-(

Comment (JamesSingleton) [reply packages/holocron/README.md]: There wasn't any there to begin with

Comment (JamesSingleton) [reply packages/holocron/README.md]: I will add these images in

**Comment by JamesSingleton** [comment]
So the reason the images are currently _broken_ is because I am pointing to the master branch instead of the `update-readme` branch. 

This URL https://raw.githubusercontent.com/americanexpress/holocron/update-readme/packages/holocron-module-route/holocron-module-route.png works, but I don't want to point to a branch that will get deleted.

**Pull request merged**